norse/benchmark/main.py

```diff
@@ -55,6 +57,13 @@ def benchmark(
             for _ in range(config.runs):
                 duration = model(parameters)
                 durations.append(duration)
+                # Clean up by GC and empty cache
+                # Thanks to https://github.com/BindsNET/bindsnet/issues/425#issuecomment-721780231
+                gc.collect()
```

Comment:
I'm not sure if this is necessarily necessary or representative.

Reply:
Unfortunately that's very hard to provide a clear answer to. It's normal in benchmarks that you 'prepare' the memory to provide as clean a slate as possible. Having a forced garbage collection does that. Whether it helps. Good question. But all in all, it should make it a tad more generalisable.